mmpose/models/utils/rtmpose_block.py

```diff
@@ -0,0 +1,248 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn.bricks import DropPath
+
+
+def rope(x, dim):
+    """Rotary Position Embedding."""
```

Comment:
This function may need a more detailed docstring on the embedding process and function signature.

Reply:
done.